setup.py

```diff
@@ -6,27 +6,24 @@
 here = os.path.abspath(os.path.dirname(__file__))
 
 setup(
-    name='easydict',
-    version='1.10.dev0',
-    author='Mathieu Leplatre',
-    author_email='mathieu.leplatre@makina-corpus.com',
-    url='https://github.com/makinacorpus/easydict',
-    download_url="http://pypi.python.org/pypi/easydict/",
-    description="Access dict values as attributes (works recursively).",
+    name='monkeydict',
+    version='1.0',
+    author='Mathieu Leplatre, Jeremiah Blanchard',
+    url='https://github.com/cacticouncil/monkeydict',
+    download_url="http://pypi.python.org/pypi/monkeydict/",
+    description="Access dict values as attributes (works recursively). Fork of EasyDict.",
```

Comment:
It looks like you submitted your fork

Reply:
Yeah, I changed it to a fork because I needed an updated version, but we hadn't seen any activity. :) I forgot that the fork elements would propagate over here. I will update shortly.